editor/store/effects.js

```diff
@@ -499,4 +502,31 @@ export default {
 	SELECT_BLOCK: removeProvisionalBlock,
 
 	MULTI_SELECT: removeProvisionalBlock,
+
+	REMOVE_BLOCKS( action, { getState, dispatch } ) {
+		// if we are removing the provisional block don't do anything.
+		if ( action.isProvisionalBlock ) {
```

Comment:
Is it not possible to use `getProvisionalBlockUID` ? Ideally we don't have options in the action like this if we can avoid it.

Reply:
Unfortunately no, because we don't keep the history of provisionalBlockUID and when the effects code executes the provisionalBlockUID is already removed so we can not access it.